php/libraries/NDB_Page.class.inc

```diff
@@ -586,5 +586,44 @@ class NDB_Page
         }
         return $files;
     }
+
+
+    /**
+     * Returns an ordered list of CSS dependencies that this page depends
+     * on.  These will get loaded into the template in order, so that
+     * interdependent files can be included in the correct order.
+     *
+     * @return array of strings with URLs to retrieve CSS resources
+     */
+    function getCSSDependencies()
+    {
+        $factory = NDB_Factory::singleton();
+        $config  = $factory->config();
+
+        $www     = $config->getSetting('www');
```

Comment:
I had just based it on the getJSDependencies() code directly above.. but, in fact, you're right: it should be using $factory->settings()->getBaseURL();